src/db.ts

```diff
@@ -148,6 +148,11 @@ export class Db {
     // Filter the options
     options = filterOptions(options, DB_OPTIONS_ALLOW_LIST);
 
+    // Ensure there are no dots in database name
+    if (typeof databaseName === 'string' && databaseName.indexOf('.') !== -1) {
+      throw new MongoAPIError(`database names cannot contain the character '.'`);
```

Comment:
Should probably be a MongoInvalidArgument error to match what it was before.

Reply:
Sounds good, I'll add this into my next push.